docs/04.guides/04.cookbooks/33.Hidden_gems/page.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Hidden Gems
+id: hidden_gems
+---
+## Hidden Gems ##
+
+This document explains how to declare a variables, function call with dot and bracket notation, Passing arguments via URL/form scopes as array. Explain this three with simple example below:
+
+#### Example 1 : Declare variables ####
+
+// test.cfc
+
+```luceescript
+component {
+	function getName() {
+		return "Susi";
+	}
+}
+```
+
+// example1.cfm
+
+```luceescript
+function test() {
+	var qry;
+	dump(qry);
+	query name="qry" datasource="test" {
+		echo("select 1 as one");
+	}
+	dump(qry);
+}
+test();
+```
+
+* In cfm page have test() function with local variable scope, it normally assigned in empty string ``var qry``. Then execute this cfm, the qry is returns "1". 
+* Then dump the ``qry`` below of the var declaration. It returns empty string. 
+
+
+#### Example 2 : Dot and bracket notation for function calls #### 
+
+Lucee allow you to use bracket notations to call a component function. 
```

Comment:
***ToChange***
Lucee allows you to use bracket notations to call a component function.